production/tanka/grafana-agent/grafana-agent.libsonnet

```diff
@@ -53,4 +54,67 @@ k + config {
     daemonSet.mixin.spec.template.spec.withServiceAccount($._config.agent_cluster_role_name) +
     config_hash_mixin +
     $.util.configVolumeMount($._config.agent_configmap_name, '/etc/agent'),
+
+
+  local agent_deployment_configmap_name = $._config.agent_configmap_name + '_deployment',
+  // If running on GKE, you cannot scrape API server pods, and must
+  // instead scrape the API server service endpoints. On AKS this doesn't
+  // work.
+  agent_deployment_config_map:
+    if $._config.agent_host_filter then
+      configMap.new(agent_deployment_configmap_name) +
+      configMap.withData({
+        'agent.yml': $.util.manifestYaml($._config.agent_config {
+          configs: [{
+            name: 'agent',
+            host_filter: $._config.agent_host_filter,
```

Comment:
This should be explicitly set to `false` (or just removed, since `false` is the default here)

Reply:
I thought about this but are my eyes deceiving me or do I see the default as true?

Reply:
You're totally right, it's true by default in the jsonnet but false by default in the code. i.e., if we don't specify this line at all it'll defer to the code's default

Reply:
I'd rather be explicit at the expense of a bit more complexity in the code - let me know if you agree or if you'd like me to take another angle.

Reply:
I'm on board with being explicit 👍